test_diffengine.py

```diff
@@ -1,17 +1,45 @@
+import logging
 import os
 import re
+
+import yaml
+from selenium import webdriver
+
 import setup
 import pytest
 import shutil
 
-from diffengine import *
```

Comment:
Thank you for addressing this! I never should have done an `import *` here.

Reply:
It has been automatically done by the IDE (I use IntelliJ)